Cargo.toml

```diff
@@ -19,6 +19,9 @@ serde = { version = "1.0.183", features = ["derive"] }
 serde_json = "1.0.104"
 sha1 = "0.10.5"
 tokio = { version = "1.31.0", features = ["full"] }
+tower-http = { version = "0.4.3", features = ["trace"] }
+tracing = "0.1.37"
```

Comment:
![image](https://github.com/riichi/trello-to-discord-webhook-service/assets/3128220/32c6fe05-1106-4dc6-811a-95a6090d3aa6)